examples/PyOpenCL/test_build_dir

```diff
@@ -3,7 +3,12 @@
 # Runs the PyOpenCL tests against a pocl build dir located in the
 # current working directory. Assumes an ICD build in PWD.
 
-pyopencl_root=`dirname $0`
+pyopencl_root=$1
+if [ -z "$pyopencl_root" ]; then
+  pyopencl_root=`dirname $0`
+fi
+
+if [ -z "$OCL_ICD_VENDORS" ]; then
```

Comment:
current build dir of what ? pocl, or examples ? i only have one builddir of examples on my buildbots & working pc, but i have tons of pocl builddirs. A script that expects pocl builddir = example builddir is unusable for me. Also, if i set the OCL_ICD_VENDORS before running ctest (not accidentally, i do that on purpose ;), i don't want it to silently override it to whatever it wants.